.github/workflows/ci.yml

```diff
@@ -139,6 +139,7 @@ jobs:
     timeout-minutes: ${{ vars.TIMEOUT_MINUTES_LONG && fromJSON(vars.TIMEOUT_MINUTES_LONG) || 15 }}
 
     strategy:
+      fail-fast: false
```

Comment:
Given the flakiness of our integration tests, it feels valuable to continue running tests even if others fail. Many times tests are minutes into running when cancelled, which I don't think has the cost savings we expect in practice.